### PR TITLE
foxtrotgps

### PR DESCRIPTION
**Pull request by wucke13**
###### Motivation for this change
Replacing the a long time ago abandoned `tangogps` by a recent `foxtrotgps` 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by markuskowa** [comment]
An entry in `pkgs/top-level/aliases.nix` would be nice.

**Comment by wucke13** [comment]
@markuskowa

File: pkgs/applications/misc/foxtrotgps/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    description = ''
+      An easy to use, free & open-source GPS/GIS application that works well on
+      small screens, and is especially suited to touch input. It spun off of
+      tangoGPS in 2010 with a focus on cooperation and fostering community
+      innovation.
+    '';
+    homepage = "https://www.foxtrotgps.org/";
+    license = licenses.gpl2;
+    platforms = platforms.unix;
+    maintainer = with maintainers; [ wucke13 ];
```
Comment (markuskowa) [inline pkgs/applications/misc/foxtrotgps/default.nix]:
```suggestion
    maintainers = with maintainers; [ wucke13 ];
```

File: pkgs/applications/misc/foxtrotgps/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    gtk2.dev
+    intltool
+    libexif
+    sqlite.dev
+    (python3Packages.python.withPackages (pythonPackages: with python3Packages;
+    [ beautifulsoup4 feedparser sqlalchemy ]))
+    ];
+
+  meta = with stdenv.lib; {
+    description = ''
```
Comment (markuskowa) [inline pkgs/applications/misc/foxtrotgps/default.nix]: Maybe that would look better in the online pacakge list (https://nixos.org/nixos/packages.html)?
```suggestion
    description = "GPS/GIS application optimized for small screens"
    longDescription = ''
```

**Comment by markuskowa** [comment]
@GrahamcOfBorg build foxtrotgps

**Pull request merged**